src/Common/Common.Api/Gateways/GatewayClient.cs

```diff
@@ -219,10 +347,10 @@ private async Task<T> DeserializeResponseContent<T>(HttpResponseMessage response
             serializer.Converters.Add(new StringEnumConverter());
 
             using (var sr = new StreamReader(await response.Content.ReadAsStreamAsync()))
-            using (var jsonTextReader = new JsonTextReader(sr))
-            {
-                return serializer.Deserialize<T>(jsonTextReader);
-            }
+                using (var jsonTextReader = new JsonTextReader(sr))
```

Comment:
inline with above 'using'?

Reply:
VS recommends indenting it. I could wrap it in {} just to be clear.